<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBtnAssist</class>
 <widget class="QWidget" name="FormBtnAssist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>318</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>624</width>
    <height>318</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>624</width>
    <height>318</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>-1</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">注塑机动作</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 28px &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutForm">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameBtnAssist">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="IsMainArea" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelDelay">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>时间(秒)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelInterval">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>间隔(模)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBoxOil">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xIndicatorButton" name="radioBtnOil">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>打油关</string>
          </property>
          <property name="iconOn" stdset="0">
           <iconset resource="xPad.qrc">
            <normaloff>:/img/widget/buttonOn.png</normaloff>:/img/widget/buttonOn.png</iconset>
          </property>
          <property name="iconOff" stdset="0">
           <iconset resource="xPad.qrc">
            <normaloff>:/img/widget/buttonOff.png</normaloff>:/img/widget/buttonOff.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xDoubleSpinBox" name="BoxDelayOil">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string notr="true"/>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string notr="true"/>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>3600.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="HasBtn" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xSpinBox" name="BoxIntervalOil">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string notr="true"/>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBoxConveyor">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xIndicatorButton" name="radioBtnConveyor">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>传送带关</string>
          </property>
          <property name="iconOn" stdset="0">
           <iconset resource="xPad.qrc">
            <normaloff>:/img/widget/buttonOn.png</normaloff>:/img/widget/buttonOn.png</iconset>
          </property>
          <property name="iconOff" stdset="0">
           <iconset resource="xPad.qrc">
            <normaloff>:/img/widget/buttonOff.png</normaloff>:/img/widget/buttonOff.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xDoubleSpinBox" name="BoxDelayConveyor">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string notr="true"/>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string notr="true"/>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>3600.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="HasBtn" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xSpinBox" name="BoxIntervalConveyor">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string notr="true"/>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBoxReject">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xIndicatorButton" name="radioBtnReject">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>不良品</string>
          </property>
          <property name="iconOn" stdset="0">
           <iconset resource="xPad.qrc">
            <normaloff>:/img/widget/buttonOn.png</normaloff>:/img/widget/buttonOn.png</iconset>
          </property>
          <property name="iconOff" stdset="0">
           <iconset resource="xPad.qrc">
            <normaloff>:/img/widget/buttonOff.png</normaloff>:/img/widget/buttonOff.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xDoubleSpinBox" name="BoxDelayReject">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string notr="true"/>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string notr="true"/>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>3600.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="HasBtn" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="xSpinBox" name="BoxIntervalReject">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string notr="true"/>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>8</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xSpinBox</class>
   <extends>QSpinBox</extends>
   <header>xspinbox.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>xDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>xdoublespinbox.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>xIndicatorButton</class>
   <extends>QPushButton</extends>
   <header>xindicatorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="xPad.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BoxDelayOil</sender>
   <signal>clicked()</signal>
   <receiver>FormBtnAssist</receiver>
   <slot>inputDelayOil()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioBtnOil</sender>
   <signal>clicked()</signal>
   <receiver>FormBtnAssist</receiver>
   <slot>btnOilclicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioBtnConveyor</sender>
   <signal>clicked()</signal>
   <receiver>FormBtnAssist</receiver>
   <slot>btnConveyorclicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BoxDelayConveyor</sender>
   <signal>clicked()</signal>
   <receiver>FormBtnAssist</receiver>
   <slot>inputDelayConveyor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BoxDelayReject</sender>
   <signal>clicked()</signal>
   <receiver>FormBtnAssist</receiver>
   <slot>inputDelayReject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioBtnReject</sender>
   <signal>clicked()</signal>
   <receiver>FormBtnAssist</receiver>
   <slot>btnRejectclicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BoxIntervalConveyor</sender>
   <signal>clicked()</signal>
   <receiver>FormBtnAssist</receiver>
   <slot>inputIntervalConveyor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BoxIntervalOil</sender>
   <signal>clicked()</signal>
   <receiver>FormBtnAssist</receiver>
   <slot>inputIntervalOil()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BoxIntervalReject</sender>
   <signal>clicked()</signal>
   <receiver>FormBtnAssist</receiver>
   <slot>inputIntervalReject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputDelayOil()</slot>
  <slot>inputDelayConveyor()</slot>
  <slot>inputDelayReject()</slot>
  <slot>btnOilclicked()</slot>
  <slot>btnConveyorclicked()</slot>
  <slot>btnRejectclicked()</slot>
  <slot>inputIntervalOil()</slot>
  <slot>inputIntervalConveyor()</slot>
  <slot>inputIntervalReject()</slot>
 </slots>
</ui>
